avocado/utils/archive.py

```diff
@@ -175,8 +175,8 @@ def _update_zip_extra_attrs(self, dst_dir):
         for path, info in self._engine.NameToInfo.iteritems():
             dst = os.path.join(dst_dir, path)
             if not os.path.exists(dst):
-                LOG.warn("Paths in this zip file are stored in unsupported "
-                         "format, not updating the attributes. (%s)", msg)
+                LOG.warn("The zip file '%s' does not exist, "
+                         "no way to update the attributes.", dst)
```

Comment:
If I understand this correctly, `dst` refer to the full path of the file that is going to be written as part of the zip archive extraction.  That is, `dst` doesn't refer to the zip file itself.  The part about getting rid of `msg` is correct though.